ethereum-push-notification-service/epns-browser-ext-staging
# Notification page

1. **Error message on Verify Address page - Completed.**
2. Welcome Back Page for existing users, with a Link to the DAPP that redirects users to Opt-In to a Channel. (Can also guide them to docs or relevant videos, if need be) - Completed.
3. Update HomePage UI - Completed
4. Validate Correct Wallet Address - Completed
5. Update Wallet Address Page UI - Completed
6. Notify successful signup - Completed
7. Add animations and transition effects to the UI using GSAP - Completed 
8. Top Bar on extension - Completed
-Cancel button on the extension.
-EPNS Logo on the extension
9. Update notification page UI- Completed
10. Notification page illustration for new users - Completed
11. Switch account dropdown and other options to be added - Completed
12. Inbox/Spam notification - Completed

QA Observations

- Change cursor on both cancel buttons and regular buttons -DONE
- Brave Browser (size)- DONE
- Address Page remove red line in input text field -DONE
- EPNS Logo Tool-Tip -DONE
- Check dimension in extension - DONE
- Wallet Address Text Colour - DONE
- Circular border on cancel button -DONE
- Extension not opening beyond home page in Brave browser - DONE (Users need to check their settings, under ‘privacy & security’ and Enable ‘**Use Google services for push messaging**’ and relaunch their brave browser to be able to access the full functionality of the browser on brave browser)
